Description: Build with system zlib instead of embedded one
Author: Ludovic Drolez <ldrolez@debian.org>

--- a/MANIFEST
+++ b/MANIFEST
@@ -47,5 +47,3 @@
 zlib/zlib.h
 zlib/zutil.c
 zlib/zutil.h
-META.yml                                 Module YAML meta-data (added by MakeMaker)
-META.json                                Module JSON meta-data (added by MakeMaker)
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -21,11 +21,11 @@
     VERSION_FROM      => 'lib/BackupPC/XS.pm', # finds $VERSION
     ABSTRACT_FROM     => 'lib/BackupPC/XS.pm', # finds ABSTRACT
     PREREQ_PM         => {}, # e.g., Module::Name => 1.1
-    LIBS              => [''], # e.g., '-lm'
+    LIBS              => ['-lz'], # e.g., '-lm'
     DEFINE            => '', # e.g., '-DHAVE_SOMETHING'
     INC               => '-I.', # e.g., '-I. -I/usr/include/other'
     OBJECT            => $object,
-    MYEXTLIB          => 'zlib/libzlib$(LIB_EXT) md5/libmd5$(LIB_EXT)',
+    MYEXTLIB          => 'md5/libmd5$(LIB_EXT)',
     depend            => { 'BackupPC_XS.o' => 'config.h' },
     clean             => { FILES => "config.log config.status config.h" },
 );
@@ -42,8 +42,8 @@
 sub MY::postamble {
         return <<'EOF';
 
-zlib/libzlib$(LIB_EXT): zlib/Makefile config.h
-	cd zlib && $(MAKE) $(PASSTHRU)
+#zlib/libzlib$(LIB_EXT): zlib/Makefile config.h
+#	cd zlib && $(MAKE) $(PASSTHRU)
 
 md5/libmd5$(LIB_EXT): md5/Makefile config.h
 	cd md5 && $(MAKE) $(PASSTHRU)
